RobotCode/src/main/java/frc/robot/constants/ArmConstants.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.constants;
+
+public class ArmConstants
+{
+
+	public static final double ARM_LOWER_KP = 0.911;
```

Comment:
Please set all constants to 0 before testing. Also just remember to remove the upper/lower stuff here as well.